css/mobile_first.css

```diff
@@ -512,41 +515,35 @@ h3 {
 .contact textarea {
   font-size: 15px;
   line-height: 24px;
-  color: #b3bac5;
+  color: #42526e;
   padding: 15px;
   border-radius: 12px;
-  border: none;
   border: 0.5px solid #60c095;
 }
 
-.contact-socials button {
-  width: 153px;
-  padding-top: 12px;
-  padding-bottom: 12px;
+#contact-button {
```

Comment:
- [ ] As suggested by the previous reviewer I kindly suggest that you add the correct different button interactions including hover and active (when it's pressed) as shown in the Figma template below:

![image](https://user-images.githubusercontent.com/52934128/219883731-3d32d6f5-53ac-4128-a0c8-1a7595de03e5.png)

the button on the pressed state has the `background: #008552;` and on hover has the `background: #36B37F;` not `background: #033700;` as you currently have it.

### EXAMPLE:
**for pressed state**
```
button:active {
   background-color: #008552;
}
```
**for hover state**
```
button:hover {
    background-color: #36B37F;
}
```